mlrun/artifacts/manager.py

```diff
@@ -129,7 +129,7 @@ def artifact_list(self, full=False):
     def log_artifact(
         self,
         producer,
-        item,
+        item: Union[str, Artifact],
```

Comment:
Again, here you're blocking usage with `LegacyArtifact` objects. We should consider if this is valid at this point in time (we haven't completely deprecated it yet).

Reply:
I'll add a support for `LegacyArtifact`s as well.